play/src/main/resources/evalution/default/2.sql

```diff
@@ -0,0 +1,79 @@
+# --- !Ups
+
+CREATE TABLE IF NOT EXISTS Users (
```

Comment:
Hmmm... so we should still keep the 1.sql to be tracked by slick-evolution as history?